crAPI/deploy/k8s/gcp/mailhog/deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         runAsGroup: 0
       containers:
         - name: mailhog
-          image: mailhog/mailhog
+          image: jcalonso/mailhog
```

Comment:
Can we use official images? If not, would it be possible to build this image ourselves?

Reply:
It isn't. 
But it's from a fork and has a million+ pulls on dockerhub.
Picked it up from here: https://github.com/mailhog/MailHog/issues/359
We've been using it for the past couple of months now.